src/style.css

```
@@ -1,3 +1,109 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --hover-color: rgba(0, 172, 220, 1);
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.nav__container ul {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__container li {
+  margin-right: 20px;
+}
+
+.nav__container li:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  cursor: pointer;
+  justify-content: center;
+  height: 100%;
+}
+
+.nav__link:hover,
+.nav__container li:first-child .nav__link {
+  color: var(--hover-color);
+}
+
+.nav__container li:first-child {
+  margin-left: 0;
+}
+
+.nav__link.is-active::after {
```

Use a simple selector everywhere without a combination selectors
```suggestion
.is-active::after {
```